src/Plugin/RulesExpression/ActionSet.php

```diff
@@ -27,7 +27,17 @@ protected function allowsMetadataAssertions() {
    * {@inheritdoc}
    */
   public function executeWithState(ExecutionStateInterface $state) {
+    // Get hold of actions.
+    // @todo See if we can add getExpressions method of ExpressionContainerBase.
+    $actions = [];
```

Comment:
Hmm...looks like an array to me:
```
  /**
   * List of actions that will be executed.
   *
   * @var \Drupal\rules\Engine\ActionExpressionInterface[]
   */
  protected $actions = [];

  ...

  public function __construct(array $configuration, $plugin_id, $plugin_definition, ExpressionManagerInterface $expression_manager) {

    ...

    foreach ($configuration['actions'] as $action_config) {
      $action = $expression_manager->createInstance($action_config['id'], $action_config);
      $this->actions[] = $action;
    }
  }
```
But at any rate, if it's part of the other PR, then I won't split hairs here.